components/director/internal/domain/api/service_test.go

```diff
@@ -314,7 +470,18 @@ func TestService_SetAPIAuth(t *testing.T) {
 			ExpectedErr:         nil,
 		},
```

Comment:
This test still misses scenario where there is no existing auth for specific runtime.